### ISSUE TITLE
Grayscale fill() or background() with alpha value crashes sketch

### ISSUE THREAD

**Issue by catarak** [processing/p5.accessibility#20]
#### Nature of issue?

- [x] Found a bug
- [ ] Existing feature enhancement
- [ ] New feature request

#### Details about the bug: 
Originally from processing/p5.js-web-editor#590

- Web browser and version: 64.0.3282.186 (Official Build) (64-bit) (cohort: Stable)
- Operating System: Windows 10 1709
- Steps to reproduce this:

```javascript
function setup() {
  createCanvas(400, 400);
}

function draw() {
  background(0, 10);
}
```

```javascript
function setup() {
  createCanvas(400, 400);
}

function draw() {
  background(0);
  fill(255, 255)
  ellipse(mouseX, mouseY, 10, 10)
}
```
After a few frames, the sketch stops working (it freezes) and I get `Script error. (: line 0)` in the console. This only happens in Editor mode, in Fullscreen mode, it works fine. This even happens when the alpha value is 255 or 0.

The following error appears in the JS console:
```
Uncaught TypeError: Cannot read property 'color' of undefined
    at BackgroundEntity.populate (https://cdn.rawgit.com/MathuraMG/p5-accessibility/298ccdf6/dist/p5-accessibility.js:1059:68)
    at new BackgroundEntity (https://cdn.rawgit.com/MathuraMG/p5-accessibility/298ccdf6/dist/p5-accessibility.js:1062:8)
    at gridInterceptor.populateObject (https://cdn.rawgit.com/MathuraMG/p5-accessibility/298ccdf6/dist/p5-accessibility.js:1524:5)
    at e.p5.(anonymous function) [as background] (https://cdn.rawgit.com/MathuraMG/p5-accessibility/298ccdf6/dist/p5-accessibility.js:1658:25)
    at draw (about:srcdoc:55:3)
    at e.redraw (https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/p5.min.js:8:27245)
    at e.p5.(anonymous function) [as redraw] (https://cdn.rawgit.com/MathuraMG/p5-accessibility/298ccdf6/dist/p5-accessibility.js:1673:27)
    at e.<anonymous> (https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/p5.min.js:7:17856)
```

**Comment by MathuraMG** [comment]
Thanks @catarak - will look into this!

**Comment by lm-n** [comment]
We should consider all possible color notations. The following are giving errors:
- [ ] three-digit hexadecimal RGB notation `fill('#fae');`
- [ ] integer RGB notation `fill('rgb(0,255,0)');`
- [ ] RGBA `fill(55,40,30,200);` 
- [ ] integer RGBA notation `fill('rgba(0,255,0,0.25)');`
- [ ] percentage RGB notation `fill('rgb(100%,0%,10%)');`
- [ ] percentage RGBA notation `fill('rgba(100%,0%,100%,0.5)');`
- [ ] H, S & B integer values`colorMode(HSB);
fill(255, 204, 100);`

**Comment by lm-n** [comment]
@MathuraMG I've moved other issues with colors to #36. #20 should be resolved in #35.
